packages/flutter_tools/test/src/android_common.dart

```diff
@@ -43,30 +43,34 @@ class FakeAndroidBuilder implements AndroidBuilder {
   }) async {}
 
   @override
-  Future<List<String>> getBuildVariants({required FlutterProject project}) async => const <String>[];
+  Future<List<String>> getBuildVariants(
+          {required FlutterProject project}) async =>
+      const <String>[];
 
   @override
   Future<String> outputsAppLinkSettings(
     String buildVariant, {
     required FlutterProject project,
-  }) async => '/';
+  }) async =>
+      '/';
```

Comment:
These auto-formatter changes are unfortunately a readability downgrade from the original. We could either revert or replace with something better:

```dart
@override
  Future<List<String>> getBuildVariants({required FlutterProject project}) {
    return Future.value(const <String>[]);
  }

  @override
  Future<String> outputsAppLinkSettings(
    String buildVariant, {
    required FlutterProject project,
  }) {
    return Future.value('/');
  }
```

Reply:
Hmmm i'll see to it then

Reply:
i'm getting Missing type annotation. now ?

Reply:
@nate-thegrate what do u think ?